dw-menu-list-item.js

```diff
@@ -167,6 +165,30 @@ class DwMenuListItem extends LitElement {
     `;
   }
 
+  get _setListItemStyle() {
+    const item = this.action;
+    let paddingLeft = this.level * 16 + 'px';
+    let styles = {};
+
+    if (item.iconColor) {
+      if (item.iconColor.startsWith('-')) {
```

Comment:
Why this needed.

Reply:
If the integrator gives a custom CSS property in iconColor.